sound/soc/sof/debug-dsp-ops.c

```diff
@@ -175,6 +183,17 @@ static ssize_t sof_dsp_ops_tester_dfs_write(struct file *file, const char __user
 	size_t size;
 	char *string;
 
+	if (!strcmp(dentry->d_name.name, "unload_fw")) {
+		string = kzalloc(count + 1, GFP_KERNEL);
+		if (!string)
+			return -ENOMEM;
+
+		size = simple_write_to_buffer(string, count, ppos, buffer, count);
+		kfree(string);
+		sof_dsp_ops_unload_firmware(sdev);
```

Comment:
you would need to invalidate all information related to the loaded firmware, like modules and stuff...

Reply:
Typically unload_firmware would be called after the DSP is put into D3 so I dont think we need to worry about modules here yet. As for additional libraries, yes I will need to handle it when  I add support for stress testing library support in a follow up.